imi/src/test/java/org/motechproject/nms/imi/it/CdrFileServiceBundleIT.java

```diff
@@ -91,13 +99,13 @@ public void testValidRequest() throws IOException, NoSuchAlgorithmException {
 
         cdrFileService.processCdrFile(new CdrFileNotificationRequest(
                         helper.obdFileName(),
-                        new FileInfo(helper.cdrSummaryFileName(), helper.summaryFileChecksum(), 0),
+                        new FileInfo(helper.cdrSummaryFileName(), helper.summaryFileChecksum(), 6),
                         new FileInfo(helper.cdrDetailFileName(), helper.detailFileChecksum(), 1))
         );
 
         try {
             long sleepyTime = 10 * 1000L;
-            getLogger().debug("Sleeping {} seconds to give a chance to @MotechListeners to catch up...", sleepyTime/1000);
+            getLogger().debug("Sleeping {} seconds to give a chance to @MotechListeners to catch up...", sleepyTime / 1000);
             Thread.sleep(sleepyTime);
```

Comment:
This seems a bit fragile... is there a programmatic way to check that the listeners executed rather than sleeping an arbitrary amount of time?